editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -484,9 +493,17 @@ const RemoteChangesBlock = () => {
   )
   const triggerUpdateAgainstGithub = React.useCallback(() => {
     if (repo != null && branch != null && commit != null) {
-      void updateProjectAgainstGithub(dispatch, repo, branch, commit)
+      void updateProjectAgainstGithub(
```

Comment:
not related, but what's with this `void` guy here?

Reply:
`updateProjectAgainstGithub` is async from memory, so if we don't add that eslint complains.

Reply:
interesting!